starter_code/express-spotify-app/views/albums.ejs

```diff
@@ -0,0 +1,13 @@
+<h1>Albums Page</h1>
+
+<div>
+  <div>
+    <% for (let i = 0; i < albums.length; i++) { %>
+      <h3><%= albums[i].name %></h3>
+        <% if (albums[i].images[0]) { %>
```

Comment:
Good call! However, the best approach is define a helper function that, if there is an image, return it, otherwise displays a placeholder. Then you can use that function in artists.ejs too.